From 854943f4c0df6ae015fc1852c9aee4ccca0008fd Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 5 Nov 2015 13:19:20 +0000
Subject: [PATCH 02/11] open-vm-tools: Add --disable-werror configure option

Packagers will normally not want the -Werror compile option as it may
break compilation depending on the platform specific warnings.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 open-vm-tools/configure.ac | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/open-vm-tools/configure.ac b/open-vm-tools/configure.ac
index ebdf17c..810555c 100644
--- a/open-vm-tools/configure.ac
+++ b/open-vm-tools/configure.ac
@@ -935,7 +935,17 @@ AC_C_VOLATILE
 
 ### General flags / actions
 CFLAGS="$CFLAGS -Wall"
-CFLAGS="$CFLAGS -Werror"
+AC_ARG_ENABLE(
+   werror,
+   AS_HELP_STRING(
+      [--disable-werror],
+      [disable compilation with -Werror]),
+   [enable_werror="$enableval"],
+   [enable_werror="yes"])
+
+if test "$enable_werror" = "yes"; then
+   CFLAGS="$CFLAGS -Werror"
+fi
 
 # -Wno-unknown-pragmas is due to gcc not understanding '#pragma ident'
 # in Xlib.h on OpenSolaris.
-- 
2.6.3

